tests/basic/pure_test.go

```diff
@@ -4238,6 +4238,248 @@ var _ = Describe("{CreateNewPoolsWhenFadaFbdaVolumeCreationInProgress}", func()
+	It(itLog, func() {
+		log.InfoD(itLog)
+		numberOfPvc := 10
+		var k8sCore = core.Instance()
+		var wg sync.WaitGroup
+
+		//Declaring SC name, namespaces and pvc prefixes and lists which are required for collection of PVC And Volume Names
+		baseScName := "base-portworx-volume-sc"
+		fadaScName := "fada-volume-sc"
+		fbdaScName := "fbda-volume-sc"
+		BaseAppNameSpace := "base-app-namespace"
+		FadaAppNameSpace := "fada-app-namespace"
+		FbdaAppNameSpace := "fbda-app-namespace"
+		max_iops := 1000
+		max_bandwidth := 1
+		//Creating Two lists to collect the volume names of both FA and FB created volumes
+		listofFadaPvc := make([]string, 0)
+		listofFbdaPvc := make([]string, 0)
+		//Creating maps with namespaces as key and list of pvc names as values
+		namespaces := []string{FadaAppNameSpace, FbdaAppNameSpace, BaseAppNameSpace}
+		namespacePVCMap := make(map[string][]string)
+		for _, ns := range namespaces {
+			namespacePVCMap[ns] = []string{}
+		}
+		//Creating map with storage class as key and namespace as value
+		scNamespaceMap := map[string]string{
+			baseScName: BaseAppNameSpace,
+			fadaScName: FadaAppNameSpace,
+			fbdaScName: FbdaAppNameSpace,
+		}
+
+		//Get The Details of Existing FA AND FB in the cluster
+		flashArrays, err := GetFADetailsUsed()
+		log.FailOnError(err, "Failed to get FA details from pure.json in the cluster")
+		flashBlades, err := GetFBDetailsFromCluster()
+		log.FailOnError(err, "Failed to get FB details from pure.json in the cluster")
+
+		stepLog := "Create storage class with max iops and max bandwidth for Normal Portworx Volumes , FADA and FBDA Pvc Deployment"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			BaseParams := make(map[string]string)
+			BaseParams["repl"] = "1"
+			BaseParams["max_iops"] = "1000"
+			BaseParams["max_bandwidth"] = "1G"
+			reclaimPolicyDelete := v1.PersistentVolumeReclaimDelete
+			bindMode := storageApi.VolumeBindingImmediate
+			// create storage class for base volumes
+			_, err := CreatePortworxStorageClass(baseScName, reclaimPolicyDelete, bindMode, BaseParams)
+			log.FailOnError(err, "Failed to create base storage class")
+			log.InfoD("Storage class [%s] for Basic is created", baseScName)
+
+			faParams := make(map[string]string)
+			faParams["repl"] = "1"
+			faParams["max_iops"] = "1000"
+			faParams["max_bandwidth"] = "1G"
+			faParams["fs"] = "ext4"
+
+			var allowVolExpansionFA bool = true
+			// create storage class for FADA volumes
+			err = CreateFlashStorageClass(fadaScName,
+				"pure_block",
+				v1.PersistentVolumeReclaimDelete,
+				faParams, nil, &allowVolExpansionFA,
+				storageApi.VolumeBindingImmediate,
+				nil)
+			log.FailOnError(err, fmt.Sprintf("Failed to create storage class [%v] ", fadaScName))
+			log.InfoD("Storage class [%s] for FADA is created", fadaScName)
+
+			fbParams := make(map[string]string)
+			fbParams["pure_export_rules"] = "*(rw)"
+			mountOptions := []string{"nfsvers=4.1", "tcp"}
```

Comment:
If we aren't going to validate mount options, can we leave them empty? (alternatively, we can create a pod and validate that the volume is mounted with the right options)